libvirt/image/Makefile

```diff
@@ -141,3 +144,12 @@ clean:
 
 .PHONY: force
 force:
+
+push:
+ifeq ($(shell virsh -c qemu:///system vol-list --pool $(LIBVIRT_POOL) | grep $(LIBVIRT_VOL_NAME)),)
+	virsh -c qemu:///system vol-create-as --pool $(LIBVIRT_POOL) --name $(LIBVIRT_VOL_NAME) \
+		--capacity 107374182400 --allocation 2361393152 --prealloc-metadata \
```

Comment:
I use `--capacity 20G --allocation 2G` for some time now and i didn't hit any issues

Reply:
Cool. I will change to use those values.